src/brasil/gov/tiles/tiles/nitf.py

```diff
@@ -96,11 +96,7 @@ def alt(self):
 
         https://github.com/plonegovbr/brasil.gov.tiles/blob/caa250e9c3428d000174368321c42dae2fb48f92/src/brasil/gov/tiles/tiles/basic.py#L160
         """
-        return any((
-            self.data.get('image_description'),
-            self.data.get('description'),
-            self.data.get('title'),
-        ))
+        return self.data.get('image_description') or self.data.get('description') or self.data.get('title')
```

Comment:
why?

Reply:
This was wrong: `any` in Python [returns a boolean](https://docs.python.org/2/library/functions.html#any) and not the value of what's inside the parenthesis if anyone of them it's not `False`.